apps/portals/b2ai.standards/src/components/ChallengesCardDeck.tsx

```diff
@@ -0,0 +1,66 @@
+import healthSvg from '@/assets/health.svg?url'
+import soundSvg from '@/assets/sound.svg?url'
```

Comment:
@nickgros what is `?url` doing here?

Reply:
It's an ["Explicit URL Import"](https://vite.dev/guide/assets#explicit-url-imports) in Vite, but looking at this, I'm not sure if it's strictly necessary how I used it.